### PR TITLE
[Hotfix] fix file event_driven.zh.md did't adding title anchor.

### PR DESCRIPTION
**Pull request by twentyworld**
fix some translated files without anchor titles causing the problems that links may not works.

## What is the purpose of the change
fix some translated files without anchor titles causing the problems that links may not works.
Facing a title anchor issue when translating file streaming_analytics.zh.md, the link may always useless due to no anchor issue bundled to specified title.

## Brief change log
- add anchor to streaming_analtics.zh.md file and title `Side Outputs`.

## Verifying this change
Docs, docs. So no tests should be impacted.
## Does this pull request potentially affect one of the following parts:
No
## Documentation
yes, fix the docs.

### CODE REVIEW

**Comment by twentyworld** [comment]
@klion26 would you please take a look on this PR?

**Comment by klion26** [comment]
@twentyworld thanks for the work, as we add the <a> here, could you please add the `<a>` for the other headings?

**Comment by twentyworld** [comment]
@klion26 我英文不是那么流畅。你的意思是给这个文档的所有中文标题都加上锚点是吗？我下午处理一下。

**Comment by twentyworld** [comment]
@klion26 有时间的话，可以看一下这个PR。

File: docs/learn-flink/event_driven.zh.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ public static class PseudoWindow extends
   除此之外，`processElement` 和 `onTimer` 都提供了一个上下文对象，该对象可用于与 `TimerService` 交互。
   这两个回调还传递了一个可用于发出结果的 `Collector`。
 
-<a name="The-`open()`-method"></a>
+<a name="The-open()-method"></a>
```
Comment (klion26) [inline docs/learn-flink/event_driven.zh.md:123]: 这里建议改成 `the-open-method` ，[英文中的链接](http://localhost:4000/learn-flink/event_driven.html#the-open-method)也是这样，这里建议保持和英文的一致

File: docs/learn-flink/event_driven.zh.md [hunk 2]
```diff
@@ -142,7 +142,7 @@ public void open(Configuration conf) {
 实际上，如果 Watermark 延迟比窗口长度长得多，则可能有多个窗口同时打开，而不仅仅是两个。
 此实现通过使用 `MapState` 来支持处理这一点，该 `MapState` 将每个窗口的结束时间戳映射到该窗口的小费总和。
 
-<a name="The-`processElement()`-method"></a>
+<a name="The-processElement()-method"></a>
```
Comment (klion26) [inline docs/learn-flink/event_driven.zh.md:145]: 这里建议改成 `the-processelement-method` 理由同上

File: docs/learn-flink/event_driven.zh.md [hunk 3]
```diff
@@ -183,7 +183,7 @@ public void processElement(
 * 本例使用一个 `MapState`，其中 keys 是时间戳（timestamp），并为同一时间戳设置一个 Timer。
   这是一种常见的模式；它使得在 Timer 触发时查找相关信息变得简单高效。
 
-<a name="The-`onTimer()`-method"></a>
+<a name="The-onTimer()-method"></a>
```
Comment (klion26) [inline docs/learn-flink/event_driven.zh.md:186]: 这里建议改成 `the-ontimer-method` 理由同上

File: docs/learn-flink/event_driven.zh.md [hunk 4]
```diff
@@ -183,7 +183,7 @@ public void processElement(
...
 #### `onTimer()` 方法
```
Comment (klion26) [inline docs/learn-flink/event_driven.zh.md:187]: 另外帮忙把 239 行的 `name="Example"` 改成 `name=example-1` 这样 1）可以和英文版的同步；2）避免同一个页面中有两个 `Example` 的标签

Comment (twentyworld) [reply docs/learn-flink/event_driven.zh.md:187]: sure， 都fix掉了，有时间可以看一下

**Comment by twentyworld** [comment]
@klion26 如果符合合并条件了，
merge工作是你们做还是提交者来做（你们只负责approve）？如果是你们做的话，那rebase呢？

**Comment by wuchong** [comment]
Thanks @klion26 , @twentyworld ,  LGTM, merging...

**Pull request merged**